components/lib/crash/src/main/java/mozilla/components/lib/crash/CrashReporter.kt

```diff
@@ -139,11 +140,19 @@ class CrashReporter(
      * Submit a caught exception report to all registered services.
      */
     override fun submitCaughtException(throwable: Throwable): Job {
-        logger.info("Caught Exception report submitted to ${services.size} services")
+        /*
+         * if stacktrace is empty, replace throwable with invalid parameter exception so we can
+         * figure out which module is submitting caught exception reports without a stacktrace.
+         */
+        var reportThrowable = throwable
+        if (throwable.stackTrace.isEmpty()) {
+            reportThrowable = InvalidParameterException()
```

Comment:
* If this is for debugging purposes: Maybe create a custom exception for this case?
* Let's also add the original throwable as `cause` of the new exception, so that we get the original one too?

Reply:
Sure, I'll update.  Thanks,